File: UI/src/main/java/ca/warp7/rt/java/media/MediaFeature.java
package ca.warp7.rt.java.media;

import ca.warp7.rt.core.ft.Feature;
import ca.warp7.rt.core.ft.FeatureItemTab;

import java.util.Collections;
import java.util.List;

public class MediaFeature implements Feature {

    @Override
    public List<FeatureItemTab> getLoadedTabs() {
        return Collections.singletonList(new FeatureItemTab(
                "External Media", "fas-link:16:gray", getFeatureId(),
                FeatureItemTab.Group.SingleTab, ""
        ));
    }

    @Override
    public String getFeatureId() {
        return "media";
    }
}

File: UI/src/main/kotlin/ca/warp7/rt/ext/media/MediaFeature.kt
package ca.warp7.rt.ext.media

import ca.warp7.rt.core.feature.Feature
import ca.warp7.rt.core.feature.FeatureItemTab
import ca.warp7.rt.core.feature.FeatureLink

class MediaFeature : Feature {
    override val link = FeatureLink("External Media", "fas-link", 16)

    override val loadedTabs
        get() = listOf(FeatureItemTab(
                "External Media", "fas-link:16:gray", featureId,
                FeatureItemTab.Group.SingleTab, ""
        ))

    override val featureId get() = "dataset"
}
